components/ui-shell/settings.ts

```diff
@@ -0,0 +1,149 @@
+export interface NavItem {
+  label: string;
+  url?: string;
+  children?: NavItem[];
+  segment?: SegmentData;
+}
+
+export interface TopLevelNavItem extends NavItem {
+  isMegaMenu?: boolean;
+}
+
+export interface SegmentData {
+  cta: string;
+  location: string;
+}
+
+export enum Variant {
+  DEFAULT = '',
+  WITH_ACCOUNT = 'with-account',
+}
+
+export const NAV_ITEMS: TopLevelNavItem[] = [
+  {
+    label: 'Overview',
+    url: 'https://qiskit.org/overview/',
+  },
+  {
+    label: 'Learn',
+    url: 'https://qiskit.org/learn/',
+  },
+  {
+    label: 'Community',
+    children: [
+      {
+        label: 'Events',
+        url: 'https://qiskit.org/events/',
+      },
+      {
+        label: 'Advocates',
+        url: 'https://qiskit.org/advocates/',
+      },
+    ],
+  },
+  {
+    label: 'Documentation',
+    isMegaMenu: true,
+    url: 'https://qiskit.org/documentation/',
```

Comment:
Does the “Documentation” tab still linked to the homepage or it only expands to show the top menu?